app/src/ui/history/commit-list.tsx

```diff
@@ -188,22 +188,16 @@ export class CommitList extends React.Component<ICommitListProps, {}> {
     return undefined
   }
 
-  private onSelectedRangeChanged = (
-    start: number,
-    end: number,
-    source: SelectionSource
-  ) => {
-    // if user selects a range top down, start < end.
-    // if user selects a range down to up, start > end and need to be inverted.
-    // .slice is exclusive of last range end, thus + 1
-    const rangeStart = start < end ? start : end
-    const rangeEnd = start < end ? end + 1 : start + 1
-    // We reverse because we want the commits to be in ascending order.
-    // First commit in history -> first on array
-    const commitSHARange = this.props.commitSHAs
-      .slice(rangeStart, rangeEnd)
-      .reverse()
-    const selectedCommits = this.lookupCommits(commitSHARange)
+  private onSelectionChanged = (rows: ReadonlyArray<number>) => {
+    // Multi select can give something like 1, 5, 3 depending on order that user
+    // selects. We want to ensure they are in chronological order for best
+    // cherry-picking results. If user wants to use cherry-picking for
+    // reordering, they will need to do multiple cherry-picks.
+    // Goal: first commit in history -> first on array
+    const sorted = rows.map(r => r).sort((a, b) => (a - b) * -1)
```

Comment:
Why not just this? 🤔 
```suggestion
    const sorted = rows.map(r => r).sort((a, b) => b - a)
```

Reply:
Also, why the `rows.map(r => r)` part? 🤔 is that to create a copy?

Reply:
`b - a` because that is not how my brain thinks inversely 😁 

`rows.map(r => r)` does copy it but for my purposes converts it into a normal not readonly array that `sort` can mutate. I could have modified the `List` component to just pass a non-readonly array, but didn't think my specific problem warranted that change.

Reply:
For the copy, I was thinking about something more explicit like a `rows.slice()` like I've seen in other places of the codebase. I wish there was an ever more explicit `rows.mutableCopy()`-like method, but well 😂

Reply:
For context, discussed the copy and landed on using the spread operator. I am of the camp that `slice` is not more explicit than `map`. After research of some options, we landing on using the spread operator `[...rows]`.  

Also notes for future reference, efficiency was taking into consideration. According to [this stack overflow](https://stackoverflow.com/questions/3978492/fastest-way-to-duplicate-an-array-in-javascript-slice-vs-for-loop)  the spread operator is the fastest. Also to note `slice` is 2nd fastest far outpacing `map`. There is also `Array.From` which is similar performance to `slice` and `...`. Seems it could also be considered a good choice.